src/components/ChatLog.js

```diff
@@ -0,0 +1,36 @@
+import ChatEntry from './ChatEntry';
+import './ChatLog.css';
+import PropTypes from 'prop-types';
+
+const ChatLog = (props) => {
+  return <div className='chat-log'>
+    {props.entries.map((message) => (
+      <ChatEntry
+        sender={message.sender}
+        body={message.body}
+        timeStamp={message.timeStamp}
+        id={message.id}
+        key={message.id}
+        liked={message.liked}
+        onSetLike={props.onSetLike}
+      />
+    ))
+    }
+  </div>;
+};
+
+ChatLog.propTypes = {
+  entries: PropTypes.arrayOf(
+    PropTypes.shape({
```

Comment:
👍 Nice use of `shape` to describe the data the values within the array should have.